src/VirtualClient/VirtualClient.Core/VirtualClientComponentExtensions.cs

```diff
@@ -108,7 +113,9 @@ public static class VirtualClientComponentExtensions
             CancellationToken cancellationToken,
             bool runElevated = false,
             string username = null,
-            Action<IProcessProxy> beforeExecution = null)
+            Action<IProcessProxy> beforeExecution = null,
+            TimeSpan? timeout = null,
```

Comment:
Please let me know why this change was needed.

Reply:
As discussed, creating similar method in KafkaExecutor and removing change in VirtualClientComponentExtensions.